README.md

```diff
@@ -6,11 +6,33 @@ Please create a branch called `homework`. Then, create a file named `homework.js
 
 <ol>
   <li>Create a function that will get the sum of the numbers between 1 and n and return the answer
-  <br>
+  <br> 
+  
+  ```
+  
+  const summation= function (n) {
+  let total = 0;
+    for(let i = 1; i <= n; i++){
+      total += i;
+    }
+    return total;
+}
+
+  ```
     <code>summation(5) // should return 15 because 1+2+3+4+5=15</code>
   </li>
   <li>Create a program to get the sum of all the even numbers in a group
   <br>
+
+  ```
+  const summationEvenn=function (n){
+	let sum=0;
+	for(let a=1;a<=15; a++){
```

Comment:
you want to check all the numbers from 1 till the given argument, so you need to change `a<=15` to `a<=n`.